gravitee-rest-api-management/gravitee-rest-api-management-rest/src/test/java/io/gravitee/rest/api/management/rest/resource/auth/OAuth2AuthenticationResourceTest.java

```diff
@@ -514,6 +514,7 @@ public void shouldConnectNewUserWithGroupsMappingFromUserInfo() throws Exception
 
         // mock role search
         when(roleService.findByScopeAndName(RoleScope.ORGANIZATION, "USER")).thenReturn(Optional.of(mockRoleEntity(RoleScope.ENVIRONMENT, "USER")));
```

Comment:
I think there is also a bug here, we should have `mockRoleEntity(RoleScope.ORGANIZATION)`